RevenueCatUI/Templates/Components/Button/ButtonComponentView.swift

```diff
@@ -76,7 +76,7 @@ struct ButtonComponentView: View {
         switch destination {
         case .customerCenter:
             showCustomerCenter = true
-        case .URL(let url, let method),
+        case .url(let url, let method),
```

Comment:
Against all my nature I used all capitals here because I thought that's the Swift-way hahah. TIL enum cases in lowercase. 😄